src/test/java/BaseTest.java

```diff
@@ -1,18 +1,142 @@
+        newPlayList.click();
+        WebElement nameField = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("input[name='name']")));
+        nameField.clear();
+        nameField.sendKeys("Pluto");
+        nameField.sendKeys(Keys.ENTER);
+        }
+
+        public static boolean newPlaylistIsDisplayed(){
+        WebElement newPlaylist = wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("a[class='active']")));
+        return newPlaylist.isDisplayed();
+        }
+
+        public static void searchSong(String songName){
+        WebElement searchSong = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[type='search']")));
+        searchSong.sendKeys(songName);
+        
+        }
+        public static void clickViewAllButton(){
+        WebElement clickViewAllButton = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("button[data-test = 'view-all-songs-btn']")));
+        clickViewAllButton.click();
+        }
+
+        public static void selectFirstSong(){
+        WebElement selectFirstSong = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("section#songResultsWrapper tr.song-item td.title")));
+        selectFirstSong.click();
+        }
+
+        public static void clickAddToButton(){
+         WebElement addToButton = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("button[class = 'btn-add-to']")));
+         addToButton.click();
+        }
+
+
+    public WebElement getDeletedPlaylistMsg(){
+        return driver.findElement(By.cssSelector("div.success.show"));
+
+    }
+    public void openPlaylist() throws InterruptedException{
+        WebElement emptyPlaylist = driver.findElement(By.cssSelector(".playlist:nth-child(3)"));
+        emptyPlaylist.click();
+        Thread.sleep(2000);
+    }
+    public void deletedPlaylist() throws InterruptedException{
+       WebElement deletedPlaylistButton = driver. findElement(By.cssSelector(".btn-delete-playlist"));
+       deletedPlaylistButton.click();
+       Thread.sleep(2000);
+    }
+
+   public void choosePlaylist(){
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)"))).click();
+   }
+   public void contextClickChoosePlaylist(){
+        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".playlist:nth-child(3)")));
+        WebElement playlistElement = driver.findElement(By.cssSelector(".playlist.nth-child(3)"));
+        actions.contextClick(playlistElement).perform();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li[data-testid='playlist-context-menu-edit']"))).click();
+
+   }
+   public void enterPlaylistName(){
+        WebElement playlistInputField = driver.findElement(By.cssSelector("input[name='name']"));
+        playlistInputField.sendKeys((Keys.chord(Keys.CONTROL, "a", Keys.BACK_SPACE)));
+        playlistInputField.sendKeys(playlistName);
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226456995-920757af-2cfc-4851-b30f-c0a4274b9f9c.png)

HINT

 public void enterPlaylistName(**String playlistName**)
 
 public boolean doesPlaylistExist(**String playlistName**)
 
 
![image](https://user-images.githubusercontent.com/104761482/226457340-113a74e3-2920-4aac-8315-322e458ae714.png)